### PR TITLE
Junior tavarez - Completed UI-III

### PR DESCRIPTION
**Pull request by JuniorTavarez**

### CODE REVIEW

**Comment by JuniorTavarez** [comment]
I completed the project!

File: great-idea/css/services.css [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+
+/* Your code starts here! */
+.main-container{
+	width: 880px;
```
Comment (liam-mei) [inline great-idea/css/services.css:69]: Your CSS is really good in terms of naming convention as well achieving the effect you need without being overly specific.

File: great-idea/index.html [hunk 2]
```diff
@@ -14,10 +14,81 @@
...
+          Demand
+        </h2>
+        <p>Get Started</p>
+      </div>
+    
+    <div  class="image">
+      <img  src="img/header-img.png" alt="Image of a code snippet.">
+    </div>
+  </section>
+    <div class="br"></div>
```
Comment (liam-mei) [inline great-idea/index.html:50]: I see you made a div and gave it a br class to make the break.  There's also a `<br>` tag that might have made your life a bit easier

File: great-idea/services.html [hunk 3]
```diff
@@ -2,105 +2,125 @@
...
+<header>
+  <nav>
+    <a>Services</a>
+    <a>Product</a>
+    <a>Vision</a>
+    <a>Features</a>
+    <a>About</a>
+    <a>Contact</a>
+  </nav>
+  <div>
```
Comment (liam-mei) [inline great-idea/services.html:27]: This outer div might be unnecessary as the a tag itself is an element you can flex.

File: great-idea/services.html [hunk 4]
```diff
@@ -2,105 +2,125 @@
...
 
-Awesome thing
-Amazing thing
-Cool thing
-Great thing
+      <p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+      eleifend. Mauris euismod facilisis iaculis.</p>
 
-<img class="services-info-img" src="img/services-info.png" alt="one of our employees hard at work">
+      <div>
```
Comment (liam-mei) [inline great-idea/services.html:111]: generally `<li>` tags are wrapped in `<ul>` tags and not div tags

**Review by liam-mei — APPROVED** [review]

**Comment by liam-mei** [comment]
**MVP**
- [x] Refactor the home page layout with flexbox where inline-block was used before.
- [x] In your navigation elements, point the services `<a href=""></a>` to services.html
- [x] Services page was successfully built and looks like design file (bottom isn't aligned)
- [ ] Update the title in your newly copied services page to say Great Idea! - Services
- [x] Using the provided design file, create the services page layout using flexbox and box model.
- [x] Wrap the great idea logo in an anchor tag and point the href to index.html
--------------------------------------------------------------------------

**Additional Comments**
- [x]  Follows Lambda Git Flow
- [ ]  Frequent, clear, and descriptive commits
- [x]  Code is organized and easy to read
- [x] Code follows consistent naming conventions, language idioms, and style.
- [x] code is DRY (Don't Repeat Yourself)

--------------------------------------------------------------------------
**Stretch**

- [ ] Create another page based on one of the navigation items in addition to services.

--------------------------------------------------------------------------

Score: 2

Great job on the project!  Of all the projects I've seen so far, your page most closely resembles the design page.  Great job with the naming and structure of the HTML.

I'd like to see a few more commits and try to be more descriptive about them as well.

**Comment by liam-mei** [comment]
You're just missing one piece.  You forgot to change the title of the services page.

**Pull request merged**